test/remote/gateways/remote_mercado_pago_test.rb

```diff
@@ -394,4 +394,18 @@ def test_successful_purchase_with_3ds
     assert_equal response.params['three_ds_info']['external_resource_url'], 'https://api.mercadopago.com/cardholder_authenticator/v2/prod/browser-challenges'
     assert_include response.params['three_ds_info'], 'creq'
   end
```

Comment:
❓ Question, maybe not related to this PR but.
 Could you include a Frictionless flow test? This looks like a challenge with the three_ds_mode optional.